.github/workflows/ci.yml

```
@@ -11,9 +11,11 @@ jobs:
     name: CI
     strategy:
       matrix:
-        nextcloudVersion: [ v23.0.4, v24.0.0, master ]
+        nextcloudVersion: [ v22.2.9, v23.0.4, v24.0.0, master ]
```

```suggestion
        nextcloudVersion: [ stable22, stable23, stable24, master ]
```
It will save you some effort to adjust to latest minor releases.

---

thanks, adjusted